### PR TITLE
update readme 😈

### PR DESCRIPTION
**Pull request by jxing-brainstation**
added information about best gitflow practices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-basic readme file
\ No newline at end of file
+Ways to use GitFlow:
+
+- [] never keep your develop branch up to date with the remote
+- [] go take a shot whenever you encounter a merge conflic
```
Comment (jennyxing) [inline README.md]: you spelled conflict wrong
```suggestion
- [] go take a shot whenever you encounter a merge conflict
```

**Review by jennyxing — APPROVED** [review]
this look okay on my end. just check with the designer on the exact copy.
please also link the Jira ticket in your description.

**Review by jennyxing — CHANGES_REQUESTED** [review]
hey this PR should be merged into main

**Comment by jxing-brainstation** [comment]
> hey this PR should be merged into main

I changed it to merge into develop, can you review again and approve

**Review by jennyxing — APPROVED** [review]

**Pull request merged**